wp/resources/sample_binaries/debruijn/run_wp_16bit.sh

```diff
@@ -0,0 +1,19 @@
+# an example that compares the debruijn method of finding the LSB instead
```

Comment:
Can we add a quick comment about what the expected result (SAT vs UNSAT) is inside the run files so we don't have to look them up in the test suite when we are running them directly?